### DISCUSSION TITLE
Party tile highlight when holding staff at verzik p1

### DISCUSSION THREAD

**Discussion by k3iko** [runelite/runelite#18541] [Ideas]
As title suggests, could we get a plugin to indicate by the party orbs whos holding the staff in p1 verzik?  
At times particularly in 112 order of use, or if orb order gets messed up, It would be nice to see where the order is up to with the staff. 
Possibly just turn the orb to purple or an icon next to the orb would be great.
![purp](https://github.com/user-attachments/assets/8920e924-a070-4ffc-aaf6-53da551eefd4)

**Comment by k3iko** [comment]
🙏

**Reply by NCG-RS** [reply]
@k3iko 
https://streamable.com/22qyz6
Color for orb holders orb is configurable. There's more functionality that I'm wanting to add before I submit the plugin for approval as this is part of a bigger project, but it will be coming.

**Reply by k3iko** [reply]
This looks great! Exactly what I was after!

**Reply by k3iko** [reply]
Any update on this one?

**Reply by NCG-RS** [reply]
> Any update on this one?

This hasn't been forgotten. Had been more focused on playing than developing for the last while. There's still more that I want to add before the plugin gets released, and my main concern regarding attempting to release early with only a couple of the features is that the runelite team might prefer it get added to ToB qol rather than be a standalone plugin. I'd rather not do this given the scope of the feature set I have in mind.

I'll be looking to come back to this in the near future to wrap things up and will update here once things are complete.

**Reply by NCG-RS** [reply]
> Any update on this one?

This is being submitted for review in the next day or two. If all goes well it will be available starting next week. Really depends on how the review process goes though.

**Reply by k3iko** [reply]
looking forward to it!

**Reply by NCG-RS** [reply]
> looking forward to it!

Approval took longer than I had expected but it just went live this morning
ToB Utilities on the plugin hub

**Reply by k3iko** [reply]
Plugin looks great, thank you! Love your work
